database/mysql/mysql_user.py

```diff
@@ -320,6 +320,11 @@ def privileges_unpack(priv):
     privs = []
     for item in priv.strip().split('/'):
         pieces = item.strip().split(':')
+        dbpriv = pieces[0].rsplit(".", 1)
+        # Do not escape if privilege is for database '*' (all databases)
+        if dbpriv[0].strip('`') != '*':
+            pieces[0] = "`%s`.%s" % (dbpriv[0].strip('`'), dbpriv[1])
```

Comment:
Yeah, this is wrong. dbpriv[1] also needs to be quoted with backticks if it is not `*`.

This still breaks idempotency for the table side of the dot.

See my PR #3858